CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 ## [Unreleased]
 
+**BREAKING** Rename LPI2C.SIER field "AM1F" to "AM1IE".
+
+**BREAKING** Rename SRC.SRSR field "LOCKUP" to "LOCKUP_SYSRESETREQ" for 1011,
+1015, and 1051 MCUs.
+
+**BREAKING** Rename CCM.CS1CDR field "FLEXSPI1" to "FLEXSPI2" for the 1060
```

Comment:
I think you meant FLEXIO, this typo also occurs in the commit message. According to the diff 1051 is also affected.